vagrant/Vagrantfile

```diff
@@ -0,0 +1,26 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+
+    # Ubuntu 16.04 LTS
+    config.vm.box = "fosskers/docker-1.12"
```

Comment:
I think that we should be using a vanilla Vagrant box here vs. one that we have to maintain.